sdk/storage/azure-storage-blobs/inc/azure/storage/blobs.hpp

```diff
@@ -12,9 +12,13 @@
 #include "azure/storage/blobs/blob_client.hpp"
 #include "azure/storage/blobs/blob_container_client.hpp"
 #include "azure/storage/blobs/blob_lease_client.hpp"
+#include "azure/storage/blobs/blob_options.hpp"
+#include "azure/storage/blobs/blob_responses.hpp"
```

Comment:
I won't expect people directly depend on these two files. But I assume it's ok to add them here.

Reply:
It is easier to include all, rather than judge individually. The goal of the file is to include all public APIs. The headers are in the public include directory, so I think it is easier to include.